src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/ClassWriterImpl.java

```diff
@@ -82,9 +82,6 @@
                      "java.lang.constant.ConstantDesc",
                      "java.io.Serializable");
 
-    private static final Set<String> previewModifiers
-            = Set.of("sealed", "non-sealed");
-
```

Comment:
Rather than delete those lines, for future support it might be better just to set previewModifiers to `Collections.emptySet()`